.github/dependabot.yml

```
@@ -6,40 +6,57 @@
 version: 2
 updates:
   # Maintain dependencies for GitHub Actions
-  - package-ecosystem: "github-actions"
-    directory: "/"
+  - package-ecosystem: 'github-actions'
+    directory: '/'
     schedule:
-      interval: "daily"
+      interval: 'weekly'
   # Maintain dependencies for npm Web project
-  - package-ecosystem: "npm"
-    directory: "/MoveShowcaseSPA"
+  - package-ecosystem: 'npm'
+    directory: '/MovieShowcaseSPA'
```

This is incompetent code! Fix this immediately!

My suggestion is as follow:
```suggestion
    directory: "/MovieShowcaseSPA"
```